### DISCUSSION TITLE
The display time on the x-axis of the chart in the Dashboard shows only the date, without hours, minutes, and seconds.

### DISCUSSION THREAD

**Discussion by HGladiator** [langfuse/langfuse#3178] [Ideas]
### Describe the feature or potential improvement

![PixPin_2024-08-31_12-08-33](https://github.com/user-attachments/assets/41ed268f-9dd5-4635-aa98-923c0c6d992c)

**Comment by marcklingen** [comment]
thanks, created issue to fix this: https://github.com/langfuse/langfuse/issues/3184

**Reply by marcklingen** [reply]
fixed recently, forgot to post an update here @HGladiator, thanks again for reporting
